README.md

```diff
@@ -8,9 +8,13 @@ Current state of the software:
 
```

Comment:
We should totally update the main README mentioning this interface! (And also BayesMixPy, why not)